ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -409,8 +777,29 @@ KEXT_STATIC bool DeleteOpIsForRename(vnode_t vnode)
     return isRename;
 }
 
+template <typename EventTracer>
+    bool TraceAndTrySendRequestAndWaitForResponse(
+        EventTracer& eventTracer,
+        VirtualizationRootHandle root,
+        MessageType messageType,
+        const vnode_t vnode,
+        const FsidInode& vnodeFsidInode,
+        const char* vnodePath,
+        const char* fromPath,
+        int pid,
+        const char* procname,
+        int* kauthResult,
+        int* kauthError)
+{
+    eventTracer.SendProviderMessage(MessageType_KtoU_RecursivelyEnumerateDirectory);
```

Comment:
Do you mean messageType above?

Reply:
Yes of course, thanks for catching that!